packages/quantic/force-app/main/default/lwc/quanticHeadlessLoader/quanticHeadlessLoader.js

```diff
@@ -70,21 +82,19 @@ const getRegisteredComponent = (element, engineId) => window.coveoHeadless[engin
  * @param {string} engineId The id of the engine.
  */
 const instantiateWindowEngineObject = (element, engineId) => {
+  const newObject = {
```

Comment:
can we give it a prettier name? :D 

`newWindowEngineObject` ?

Reply:
`createWindowEngineObject` final offer

Reply:
oh you mean the const there. lol yes ok